lighthouse-core/report/templates/report.html

```diff
@@ -20,7 +20,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
-  <title>Lighthouse report: {{ url }}</title>
+  <title>Lighthouse report: {{ meta.url }}</title>
```

Comment:
should should probably be `initialUrl` based on discussion in #715. And it's not on the meta property?